TutorialApplication/OrganizationInfoViewController.swift

```diff
@@ -14,6 +14,87 @@ import UIKit
     
     static let kEmployeesOrderHasChanged = "EmployeesOrderHasChanged"
     
+    static let kOrganizationHasChanged = "OrganizationHasChanged"
+    
+    @IBAction func fetchOrganizations()
+    {
+        RequestManager.fetchOrganizations { (response) in
+        
+        var newOrganization:FFOrganization
+        
+        var newEmployee:FFEmployee
+            
+        let countOfOrganizations = DatabaseController.requestResults(for: nil, sortDescriptors: nil, entity: "FFOrganization")
+        print(countOfOrganizations?.count as Any)
+        
+        if (countOfOrganizations?.count)! <= 1
+        {
+        let organizations = response["organizations"]
+        
+        for jsonObjects in organizations as! [AnyObject]
+            {
+                let organizationName = jsonObjects["name"] as! String
+                
+                let organizationEmployees = jsonObjects["employees"] as! [[String : AnyObject]]
+                
+                print("org name = \(organizationName)  employess = \(organizationEmployees)")
+                
+                newOrganization = NSEntityDescription.insertNewObject(forEntityName: "FFOrganization", into: DatabaseController.sharedInstance().context) as! FFOrganization
+                
+                newOrganization.name = organizationName
+                
+                for item in organizationEmployees
+                {
+                    let dateFormatter = DateFormatter()
+                    dateFormatter.dateFormat = "yyyy-MM-dd"
+                    
+                    newEmployee = NSEntityDescription.insertNewObject(forEntityName: "FFEmployee", into: DatabaseController.sharedInstance().context) as! FFEmployee
+
+                    newEmployee.firstName = (item["first_name"] as! String)
+                    newEmployee.lastName = (item["last_name"] as! String)
+                    newEmployee.order = item["order"] as! Int16
+                    
+                    if item["salary"] != nil && !(item["salary"] is NSNull)
+                    {
+                        newEmployee.salary = item["salary"] as! Int32
+                    }
+                    
+                    newEmployee.isActive = item["isActive"] as! Bool
+                    newEmployee.dateOfBirth = dateFormatter.date(from:"1986-04-13")
+                    newOrganization.addEmployeesObject(newEmployee)
+                }
+                 DatabaseController.saveContext()
+            }
+        }
+            
+            
+        let actionSheetController = UIAlertController(title: "Please select organization", message: "", preferredStyle: .actionSheet)
+            
+        let cancelActionButton = UIAlertAction(title: "Cancel", style: .cancel)
+        { action -> Void in
+            print("Cancel")
+        }
+        actionSheetController.addAction(cancelActionButton)
+            
+            for organizations in 0...Int((countOfOrganizations?.count)!)-1
```

Comment:
You should first parse organizations, store them to the database, fetch organizations from the DB and present action sheet according to that data.